core/src/main/java/io/questdb/std/Rnd.java

```diff
@@ -196,6 +199,13 @@ public void nextUtf8AsciiStr(int len, Utf8Sink sink) {
         }
     }
 
+    public void shuffle(List<?> list) {
```

Comment:
Damn, this is skewed shuffle! I should use `nextInt(i + 1)` in order to get exactly `n!` options.
Will fix it later